src/App.css

```diff
@@ -26,142 +45,47 @@ input {
   align-items: center;
 }
 
-.Link {
+button, a {
```

Comment:
Heads up, this will affect all the `button` and `a` elements across the site.

For TCL, **this is fine** since it's a controlled project with a defined scope and just a few links/buttons!

This approach has a downside though, here you're treating buttons and anchors as if they were the same element, but there are situations where you want a link to look like a link, and not like a button and vice versa. Imagine what would happen if you have to create 3 different type of buttons and 4 types of links 😱.

That said, the original approach (having a `Link` class) would work better because you can do:

```html
<!-- button that looks like a link -->
<button class="Link">I am a button, but I look like a link</button>
<!-- link that looks like a link -->
<a class="Link">I am a link and I look like a link</a>
<!-- link that looks like a button -->
<a class="Button">I am a link but I look like a button<a>
<!-- and so on... -->
```

Reply:
Alejandro speaks from experience here 😅

Reply:
Yeah, I knew we didn't have any links (at least not yet) that we wanted to look like links so I went with the general elements. Using a class does make more sense though. I think that'd be a good change to make! I'll pop into the code and do that this evening 🙂

Reply:
Change made here, too! Now all are part of the `button-link` class!